### DISCUSSION TITLE
Keyboard inputs not working on android

### DISCUSSION THREAD

**Discussion by hayukimori** [ppy/osu#15565] [Q&A] [answered]
i had a problem in the last release of osu!lazer android ( sh.ppy.osulazer.apk ), after i updated, android keyboard inputs started not working.

adb inputs work

Model: LG K40S

**Comment by peppy** [comment]
Change your keyboard.

**Reply by hayukimori** [reply]
https://youtu.be/C7WizkN7_6A

To get a better look at the situation, I'm uploading a video.

**Comment by Susko3** [comment, marked as answer]
Fixed by https://github.com/ppy/osu-framework/pull/4952.
